modules/ROOT/pages/index.adoc

```diff
@@ -96,15 +100,42 @@ See the xref:api-community-manager::index.adoc[Community Manager documentation]
 [NOTE]
 Some items displayed in the Access Management UI, such as API alerts and analytics information, cannot be accessed or configured.
 
-|xref:object-store::osv2-guide.adoc[Object Store] |Share data and state across Mule runtime workers within a single application.
+|xref:mq::index.adoc[Anypoint MQ] | Send messages between different applications or components within an application.
+
+[NOTE]
+include::partial$mq-in-govcloud.adoc[tag=mqSupportInGovCloud]
+
+|xref:object-store::index.adoc[Object Store] |Share data and state across Mule runtime workers within a single application.
 |xref:api-manager::latest-overview-concept.adoc[API Manager] |Manage, govern, and secure the APIs you deploy to Anypoint Platform.
 |xref:runtime-manager::index.adoc[Runtime Manager]|Deploy, manage, and monitor your Mule applications in a sandbox, staging, or production environment from one central location, whether your apps are deployed in the cloud or on-premises.
 |xref:runtime-manager::cloudhub.adoc[CloudHub] |Deploy sophisticated cross-cloud integration applications in the cloud, create new APIs on top of existing data sources, integrate on-premises applications with cloud services, and much more.
 |xref:monitoring::index.adoc[Anypoint Monitoring] |Aggregate and map metrics across systems, configure dashboards and alerts, and store and search log data at scale.
 |xref:visualizer::index.adoc[Anypoint Visualizer] |Explore your application network for architectural review, troubleshooting, and policy compliance.
 |===
 
-=== Security in MuleSoft Government Cloud
+[[mq-in-govcloud]]
+==== Anypoint MQ in MuleSoft Government Cloud
+
+The following Anypoint MQ features are not available in MuleSoft Government Cloud:
+
+* Unencrypted queues
++
+All queues are encrypted and the *Encryption* option is not available when 
+xref:mq::mq-queues.adoc#create-a-queue[creating a queue].
+* xref:mq::mq-client-apps.adoc[Client apps]
++
+Use 
+xref:mq::mq-connected-apps.adoc[Connected Apps] instead.
+* The following REST APIs in the Anypoint MQ Stats API:
```

Comment:
Should this say "The following REST APIs in the Anypoint MQ Stats API are not available"? It currently says "The following REST APIs in the Anypoint MQ Stats API"

Reply:
Maybe the intro sentence should say something like "All features of Anypoint MQ are supported in MuleSoft Government Cloud with the following exceptions:" and then make each bullet very specific.